src/RendezvousTracker.spec.ts

```diff
@@ -383,6 +384,35 @@ describe('BrightScriptFileUtils ', () => {
         });
     });
 
+    describe('toggleEcpRendezvousTracking', () => {
+        async function doTest(statusCode: number, expectedValue: boolean) {
+            const stub = sinon.stub(util, 'httpPost').returns(Promise.resolve({ statusCode: statusCode } as any));
+            expect(
+                await rendezvousTracker.toggleEcpRendezvousTracking('track')
+            ).to.eq(expectedValue);
+            expect(
+                await rendezvousTracker.toggleEcpRendezvousTracking('untrack')
+            ).to.eql(expectedValue);
+            stub.restore();
+        }
+
+        it('returns true for 200 response code', async () => {
+            await doTest(200, true);
+            await doTest(202, true);
+            await doTest(205, true);
+            await doTest(299, true);
+        });
+
+        it('returns true for 200 response code', async () => {
```

Comment:
This name doesn't look right 😅

Reply:
Oops! Oh well, at least the tests do what I intended them to do.